### PR TITLE
Lock admin panel

### PR DESCRIPTION
**Pull request by simonovich**

### CODE REVIEW

File: spec/models/ability_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  describe "as user" do
+    let(:user) { create(:user) }
+    it "can read all" do
+      ability = Ability.new(user)
+      expect(ability).to be_able_to(:read, :all)
+    end
+  end
+  describe "as admin" do
+    it "can manage all" do
+      user = User.create(email: 'test@example.com', password: 'password123', password_confirmation: 'password123')
```
Comment (Vadser) [inline spec/models/ability_spec.rb]: Use factory-bot

File: spec/models/ability_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+RSpec.describe Ability, type: :model do
+  describe "as user" do
+    let(:user) { create(:user) }
+    it "can read all" do
+      ability = Ability.new(user)
+      expect(ability).to be_able_to(:read, :all)
+    end
+  end
+  describe "as admin" do
```
Comment (Vadser) [inline spec/models/ability_spec.rb]: Add empty line between sections

File: spec/controllers/rails_admin/main_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  routes { RailsAdmin::Engine.routes }
+
+  describe 'Check access to admin panel for User' do
+    let(:user) { create(:user) }
+
+    before do
+      sign_in user
+    end
+
+    it 'redirects to root path' do
```
Comment (anatoliliotych) [inline spec/controllers/rails_admin/main_controller_spec.rb:13]: need positive test, that admin can login

File: app/models/ability.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
```
Comment (anatoliliotych) [inline app/models/ability.rb:4]:
``` 
return unless user && user.admin?
can :access, :rails_admin
can :manage, :all 
```

**Review by anatoliliotych — CHANGES_REQUESTED** [review]

**Review by Vadser — APPROVED** [review]

File: spec/controllers/rails_admin/main_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+      get :dashboard
+#binding.pry
```
Comment (anatoliliotych) [inline spec/controllers/rails_admin/main_controller_spec.rb]: remove

File: app/views/categories/index.html.slim [hunk 6]
```diff
@@ -1,5 +1,7 @@
 p#notice
   = notice
+p#alert
```
Comment (anatoliliotych) [inline app/views/categories/index.html.slim]: move this and notice to layout

**Review by anatoliliotych — CHANGES_REQUESTED** [review]

**Review by anatoliliotych — APPROVED** [review]

**Pull request merged**